packages/components/src/Chips/ChipsTypes.tsx

```diff
@@ -96,7 +95,6 @@ export interface ChipDismissibleProps extends ChipFoundationProps {
   onLoadMore?(searchValue: string): void;
 }
 
-export type ChipsProps = XOR<
-  ChipSingleSelectProps,
-  XOR<ChipMultiSelectProps, ChipDismissibleProps>
->;
+export type ChipsProps =
```

Comment:
I didn't want to edit this, but with the XOR in place, any usage of Chips in the test files causes an error around the activator not being allowed to be undefined (even though we clearly said it was).

So instead of XOR I just used the Typescript unary operator, and explicitly set usage in Chips to single/multi/dismissible and was able to get proper type checking on my props.